packages/teleport-plugin-vue-app-routing/src/index.ts

```diff
@@ -22,14 +22,17 @@ export const createVueAppRoutingPlugin: ComponentPluginFactory<VueRouterConfig>
     dependencies.Vue = {
       type: 'library',
       path: 'vue',
+      version: '^2.6.7',
```

Comment:
Previously all these values are kind of being picked by the templates. If the templates don't have these dependencies specified. Then then these dependencies won't reflect in `package.json` even if we are using them here.